### ISSUE TITLE
Segfault on Cocopa Bay

### ISSUE THREAD

**Issue by Gymnasiast** [OpenRCT2/OpenRCT2#2685]
This is the park in question: http://www.nedesigns.com/park/3473/cocopa-bay/download/

Console log:
```
ERROR[/home/michael/Dropbox/Aptana Studio 3 Workspace/OpenRCT2-4/src/game.c:793 (game_fix_save_vars)]: Null map element at x = 15 and y = 41. Fixing...
Segmentation fault
```

**Comment by janisozaur** [comment]
There was a thread on /r/rct with this park, @IntelOrca investigated, there is a bugged ride, hiding it fixes the problem.

**Comment by devnoname120** [comment]
Would it not be possible for OpenRCT2 to automatically fix bugged rides? It could delete the invalid part, and tell the user about it.

**Comment by IntelOrca** [comment]
@devnoname120 Usually its only crashing because it can't draw it, other logic is usually fine. We simply just want to not draw it but we haven't implemented all the drawing code yet - so it crashes in original code which we can't determine the reason of very easily.

Regardless of whether we fix it or ignore drawing it, we don't know whether a thing is bugged or not because we don't know why its crashing.

**Comment by devnoname120** [comment]
Oh, OK. Has anyone experienced it when fully building a park from single player mode?

**Comment by Gymnasiast** [comment]
@devnoname120 : Most of the parks exhibiting this behaviour have been extensively modified using trainers and/or hex editing.

**Comment by janisozaur** [comment]
Should be fixed by https://github.com/OpenRCT2/OpenRCT2/pull/3020

**Comment by Gymnasiast** [comment]
Is indeed fixed by #3020. :+1:
